buildenv/docker/mkdocker.sh

```diff
@@ -459,7 +460,6 @@ if [ $arch = x86_64 ] ; then
   echo " && $wget_O gettext.tar.gz http://ftp.gnu.org/gnu/gettext/gettext-$gettext_version.tar.gz \\"
   echo " && tar -xzf gettext.tar.gz \\"
   echo " && cd gettext-$gettext_version \\"
-  echo " && ./autogen.sh --skip-gnulib \\"
```

Comment:
Please explain how it's safe to skip this step. Again, is it only relevant for centos:6?

Reply:
This step fails on the latest centos6, I don't recall all the details at this point, but removing didn't seem to cause problems with building the component.